Activities/Cryptography/UiPath.Cryptography.Activities/EncryptFile.cs

```diff
@@ -175,18 +187,47 @@ protected override void Execute(CodeActivityContext context)
                 {
                     fileName = outputFileName;
                 }
+                else
+                {
+                    fileName = Path.GetFileNameWithoutExtension(inputFilePath) + Encrypted + Path.GetExtension(inputFilePath);
```

Comment:
This block and the one in DecryptFile are functionally idential, just the arguments are different. Please consolidate them in a helper.

Reply:
fixed